clireporter/cli.go

```diff
@@ -11,29 +14,78 @@ type cli struct {
 	Executor *taskrunner.Executor
 }
 
-func Option(r *taskrunner.Runtime) {
-	r.Subscribe(func(events <-chan taskrunner.ExecutorEvent) error {
-		for event := range events {
-			switch event := event.(type) {
-			case *taskrunner.TaskInvalidatedEvent:
-				fmt.Fprintf(event.TaskHandler().LogStdout(), "Invalidating %s for %d reasons:", event.TaskHandler().Definition().Name, len(event.Reasons))
-				for _, reason := range event.Reasons {
-					fmt.Fprintf(event.TaskHandler().LogStdout(), "- %s", reason.Description())
+func StdoutOption(r *taskrunner.Runtime) {
+	option(new(os.Stdout))(r)
```

Comment:
maybe make these functions more explicit, they seem like builtins (even though they are not)

Reply:
Yeah, whoops!